website/snippets/_install-dbt-extension.md

```diff
@@ -143,12 +143,166 @@ file during registration. If you do not have a `~/.dbt/dbt_cloud.yml` file downl
 2. Click your account name at the bottom of the left-side menu and click **Account settings**.
 3. Under the **Your profile** section, click **CLI**. 
 4. In the **Configure Cloud authentication** section, click **Download CLI configuration file**. This downloads the `dbt_cloud.yml` file. 
-    <Lightbox src="/img/docs/extension/download-registration.png" width="60%" title="Download the dbt_cloud.yml file to complete registration."/>
+    <Lightbox src="/img/docs/extension/download-registration.png" width="70%" title="Download the dbt_cloud.yml file to complete registration."/>
 5. Move the downloaded `dbt_cloud.yml` file to your `~/.dbt/` directory.
 6. To update your registration in VS Code, open the command palette (`ctrl+shift+P` (Windows/Linux) or `cmd+shift+p` (macOS)), then select `dbt: Register dbt extension` to complete the registration.
 
 </Expandable>
 
+## Configure environment variables
+
+You need to configure [environment variables](/docs/build/environment-variables) (for example, [`DBT_ENV_VAR1`](/reference/dbt-jinja-functions/env_var)) in order to use the extension's features. You can configure environment variables in a couple of different ways (for example, user-level or workspace-level). 
+
+This set up will explain how to configure environment variables at the user-level. Use the following table to see which works best for you:
+
+| <div style={{width:'200px'}}>Where to configure</div> | Info | Stays active | Best for |
+|---|---|---|---|
+| [OS / shell (recommended)](#configure-at-the-os--shell-level) | Configure at the OS / shell level.<br /><br /> Affects all terminals _and_ the dbt extension (after VS Code reload) | Yes<br /> (across sessions) | Using both extension buttons _and_ terminal commands |
+| [dbt VS Code extension settings](#configure-in-the-vs-code-extension-settings) | Configure directly in the [User Settings](vscode://settings/dbt.environmentVariables) interface or in the [JSON](vscode://command/workbench.action.openSettingsJson) file.<br /><br /> Affects the dbt LSP, compile-on-save, extension menus/buttons | Yes<br /> (per VS Code settings) | Editor-only workflows |
+| [Terminal session](#configure-in-the-terminal-session) | Configure in the terminal session using the `export` command.<br /><br /> Affects the current terminal window _only_.  | No<br /> (per session) | Quick, temporary testing |
+
+:::tip Recommendation
+If you use both the dbt extension’s buttons/menus _and_ the terminal, define your env vars at the _OS/shell_ level. This works everywhere with one setup.
```

Comment:
I'm not convinced this actually works! Also, I don't love encouraging users to stick their dbt-specific env vars within their global OS user settings / shell profiles